README.md

```diff
@@ -99,18 +138,55 @@ $ sudo -E PATH=$PATH make USE_DOCKER=true image
 $ sudo -E PATH=$PATH make USE_DOCKER=true AGENT_INIT=yes image
 ```
 
+#### dracut based image
+
+> Note: the dracut build method generates an image by first building an initrd,
+and then using the rootfs extracted from it.
+
+```
+$ sudo -E PATH=$PATH make BUILD_METHOD=dracut image
+```
+
 ### Initrd creation
 
-To create an initrd from the already-created rootfs with the agent acting as the init daemon:
+#### Rootfs based initrd
+
+Create an initrd from the already-created rootfs and with the agent acting as the init daemon
+using:
 
 ```
 $ sudo -E PATH=$PATH make AGENT_INIT=yes initrd
 ```
 
+#### dracut based initrd
+
+Create an initrd using the dracut build method with:
+
+```
+$ sudo -E PATH=$PATH make BUILD_METHOD=dracut AGENT_INIT=yes initrd
+```
+
 For further details,
 see [the initrd builder documentation](initrd-builder/README.md).
 
-### Tests
+### dracut options
+
+#### Add kernel modules
+
+If the initrd or image needs to contain kernel modules, this can be done by:
+
+1. Specify the name of the modules (as reported by `modinfo MODULE-NAME`) in
+`dracut/dracut.conf.d/10-drivers.conf`. For example this file can contain:
+```
+drivers="9p 9pnet 9pnet_virtio"
+```
+2. Set the `DRACUT_KVERSION` make variable to the release name of the kernel that
+is paired with the built image or initrd, using the `uname -r` format. For example:
+```
+make BUILD_METHOD=dracut DRACUT_KVERSION=5.2.1-23-kata AGENT_INIT=yes initrd
+```
+
+## Testing
```

Comment:
Do we need to add some words to the "Platform-Distro Compatibility Matrix" section explaining that the dracut method works for all architectures I wonder?

Reply:
I'll do that.